scripts/standardising.py

```diff
@@ -100,8 +101,14 @@ def standardising(file: str, preset: str, cutline: Optional[str]) -> FileTiff:
             run_gdal(get_cutline_command(input_cutline_path), input_file=input_file, output_file=target_vrt)
             input_file = target_vrt
 
-        command = get_gdal_command(preset)
-        command.extend(get_gdal_band_offset(input_file))
+            # gdalinfo to get band offset and band type
+            info = gdal_info(file, False)
+            convert_to_byte = False
+            if get_gdal_band_type(input_file, info) == "UInt16":
+                convert_to_byte = True
+
+            command = get_gdal_command(preset, convert_to_byte)
```

Comment:
I get a little nervous passing booleans into functions, could this be a keyword argument eg `covert_to_byte=convert_to_byte` 

Makes it more clear that the boolean is very specific to this conversion.

also,  there could be a lot of "convert_x_to_byte" do we want to be a little more specific that this only works for uint16s ?

Reply:
I had this thought, but as there is currently only one conversion and I could not predict if there will be other, I preferred not being specific yet.
I'm happy to change that if it's too confusing

Reply:
> I get a little nervous passing booleans into functions, could this be a keyword argument eg covert_to_byte=convert_to_byte

Passing the input band type to `get_gdal_command(preset, band_type="UInt16")` and dealing with it in `get_gdal_command` would suit?

Reply:
I like your second option!

`convert_from` or `input_band_type` both sound like good options!